modules/ROOT/pages/_partials/gs-adding-users-proc.adoc

```diff
@@ -18,9 +18,9 @@ endif::[]
 | |*{rhmi-sso-name}* |*{customer-sso-name}*
 
 |Usage
-|Define the identities of the integration engineers and developers logging into the Red Hat applications available from the Solution Explorer.
+|Using the Solution Explorer, define the identities of the integration engineers and developers logging into the available Red Hat applications.
```

Comment:
maybe not clear, 'define the users of the cluster applications' better?

Reply:
Not sure I understand
```suggestion
Define the users of the cluster applications.
```
Is that all? Get rid of the rest?

Reply:
that does sound good, but we wanna promote SolX as the single point of entry, and there's lots of confusion about 'users', so maybe :

Create the users of the cluster applications accessible from Solution Explorer, typically developers and integration engineers.